manifests/config.pp

```diff
@@ -65,7 +65,7 @@
     class { '::foreman::config::passenger': } -> anchor { 'foreman::config_end': }
 
     if $::foreman::ipa_authentication {
-      if !defined('$::default_ipa_server') or empty($::default_ipa_server) or !$::default_ipa_realm or empty($::default_ipa_realm) {
+      if !defined('$default_ipa_server') or empty($::default_ipa_server) or !defined('$default_ipa_realm') or empty($::default_ipa_realm) {
```

Comment:
Things don't work with `::`, I have no idea why. But @ares suggested I drop them and things started to work for me, with Foreman develop.